src/ui-test/tests/reload.test.ts

```diff
@@ -0,0 +1,75 @@
+import * as path from 'path';
+import {
+    ActivityBar,
+    after,
+    before,
+    EditorView,
+    SideBarView,
+    ViewControl,
+    ViewSection,
+    VSBrowser,
+    WebDriver
+} from 'vscode-uitests-tooling';
+import {
+    CAMEL_RUN_ACTION_LABEL,
+    waitUntilTerminalHasText,
+    TEST_ARRAY_RUN,
+    executeCommand,
+    killTerminal,
+    HELLO_CAMEL_MESSAGE,
+    activateTerminalView,
+    CAMEL_ROUTE_YAML_WITH_SPACE,
+    HELLO_WORLD_MESSAGE,
+    CAMEL_ROUTE_YAML_WITH_SPACE_COPY,
+    replaceTextInCodeEditor
+} from '../utils';
+import { expect } from 'chai';
+import { createResourceManager, IResourceManager } from 'vscode-uitests-tooling/out/utils/Resources';
+import { createWorkspace } from 'vscode-uitests-tooling/out/utils/Workspace';
+
+describe('Jbang commands with automatic reload', function () {
+    this.timeout(240000);
+
+    let driver: WebDriver;
+    let resourceManager: IResourceManager;
+
+    before('Before setup', async function () {
+        driver = VSBrowser.instance.driver;
+
+        resourceManager = createResourceManager(VSBrowser.instance, createWorkspace(VSBrowser.instance, 'src/ui-test/resources'), 'src/ui-test/resources');
```

Comment:
@mlorinc 
Hi, thanks for advice according that Class
works nice do copy and delete

But I faced few issues:
1) by default it creates new folders in resources: `store` and `data` -> [gitlink](https://github.com/djelinek/vscode-uitests-tooling/blob/main/src/utils/Resources.ts#L96-L98)
2) by default `copy` use  `data` folder -> [gitlink](https://github.com/djelinek/vscode-uitests-tooling/blob/main/src/utils/Resources.ts#L125C47-L127) 

So I used `../${CAMEL_ROUTE_YAML_WITH_SPACE}` to copy/delete file
and also make extra `deleteFromStore`/`deleteStore` to delete folders due to the fact that `cleanup` deletes the whole resources folder

Reply:
Can it be issued in `vscode-uitests-tooling` to add more flexibility? Or we will adapt our folder structure to it?
WDYT? @mlorinc @djelinek

Reply:
As we discussed with @djelinek I will update `vscode-uitests-tooling`:
1) Do not create folders by default
2) By default copy from path should be `resources` not `data`
3) That PR is blocked due to that issue

Reply:
Ok, seems done